package.json

```diff
@@ -10,19 +10,19 @@
   "dependencies": {
     "@subsquid/archive-registry": "3.1.0",
     "@subsquid/graphql-server": "4.2.0",
-    "@subsquid/ss58": "1.2.0",
-    "@subsquid/substrate-processor": "^3.3.0",
+    "@subsquid/ss58": "^2.0.2",
+    "@subsquid/substrate-processor": "^8.3.1",
     "@subsquid/typeorm-migration": "1.2.0",
-    "@subsquid/typeorm-store": "1.2.1",
+    "@subsquid/typeorm-store": "^1.4.0",
     "dotenv": "^16.0.0",
-    "pg": "8.11.3",
-    "typeorm": "0.3.16"
+    "pg": "8.10.0",
```

Comment:
why downgrade postgres version?

Reply:
i believe it was conflicting with any of the peer deps of the other dependencies

Reply:
okay.. it seems to be working fine with the latest `pg` version